### ISSUE TITLE
version_decl modification causes incorrect info in compile log

### ISSUE THREAD

**Issue by davegill** [wrf-model/WRF#938]
The `compile` script has a line to inform users of the version of WRF that is being built, purely for informational purposes:
```
cat inc/version_decl | cut -c 45-54
```

While this could have been more general, it worked OK.

With the inclusion of modification 18c66c7eb99e in PR #918 "Prepare for WRF-v4.1.1 release", the string in the `inc/version_decl` file was no longer a fixed length (so the explicit character count was modified).

A modification to the `compile` script handles retrieving this information in a more robust fashion:
```
cat inc/version_decl | cut -d"'" -f2
```
which gives the correct string, regardless of the style selected for the assignment:
```
V4.1.1
```

**Comment by davegill** [comment]
@smileMchen @jamiebresch @kkeene44 @weiwangncar 
Team,
We should make this fix part of v4.1.2

**Comment by davegill** [comment]
Yep, in v4.1.2
